apps/livreGUI/tfEditor/ColorMapWidget.h

```diff
@@ -21,7 +21,7 @@
 #ifndef _ColorMapWidget_h_
 #define _ColorMapWidget_h_
 
-#include <livreGUI/tfEditor/TFWidget.h>
+#include <livreGUI/tfEditor/TFWidget.h> // base class
```

Comment:
I have the habit of giving a reason for any header which is not types/api.h for good header hygiene.